fashion-messaging-app-ui/src/components/Main/Main.jsx

```diff
@@ -59,7 +70,7 @@ function Main() {
           )}
         </div>
       </header>
-        <form className="new-post-form" onSubmit={handleSubmit}>
+        <form className="new-post-form" onSubmit={handleSubmit} encType="multipart/form-data">    
```

Comment:
The enctype allows the browser to send binary data from the image file  to the server